.idea/libraries/org_antlr_antlr4_4_5_1_1.xml

```diff
@@ -0,0 +1,12 @@
+<component name="libraryTable">
```

Comment:
Yes, that's what I was thinking. Maven is usually available as a distribution package so it shouldn't be a problem to have it on the test server.